lib/fathead/perl_doc/Parsers/parse-faq.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
```

Comment:
Yeah, basically the name of the parser sub is the value for `$parser` - I'm thinking some parsers will be useful across sets anyway, or some sets will need multiple types of parser. If we stick them in one place, it provides some mix-and-match flexibility.